### PR TITLE
make repo buildable and rename imports/images to route-controler-manager

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -11,4 +11,4 @@ approvers:
   - mfojtik
   - soltysh
   - coreydaley
```
Comment (atiratree) [inline OWNERS:13]: OWNERS file are kept in the original state.  Should we update them?

File: Makefile [hunk 2]
```diff
@@ -8,18 +8,18 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
...
+IMAGE_REGISTRY :=registry.ci.openshift.org
 
 # This will call a macro called "build-image" which will generate image specific targets based on the parameters:
 # $0 - macro name
 # $1 - target name
 # $2 - image ref
 # $3 - Dockerfile path
 # $4 - context directory for image build# It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,ocp-openshift-controller-manager,$(IMAGE_REGISTRY)/ocp/4.5:openshift-controller-manager, ./Dockerfile,.)
+$(call build-image,ocp-route-controller-manager,$(IMAGE_REGISTRY)/ocp/4.12:openshift-route-controller-manager, ./Dockerfile,.)
```
Comment (atiratree) [inline Makefile:19]: do we need the image- target built by imagebuilder?

File: Dockerfile [hunk 3]
```diff
@@ -1,10 +1,10 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.18 AS builder
-WORKDIR /go/src/github.com/openshift/openshift-controller-manager
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+WORKDIR /go/src/github.com/openshift/route-controller-manager
 COPY . .
 RUN make build --warn-undefined-variables
```
Comment (atiratree) [inline Dockerfile:4]: the build is throwing a git error: `fatal: No names found, cannot describe anything.` . It is not harmful and it is caused by having no tags. In original repo we had `v0.0.0-alpha.0` tag. Should we do something about this?

File: go.mod [hunk 4]
```diff
@@ -30,36 +18,27 @@ require (
...
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd // indirect
-	github.com/Microsoft/go-winio v0.4.17 // indirect
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/blang/semver/v4 v4.0.0 // indirect
```
Comment (deads2k) [inline go.mod:30]: hrm

File: Dockerfile (hunk 3 above)
Comment (deads2k) [reply Dockerfile:4]: we'll get a tag eventualy.

File: OWNERS (hunk 1 above)
Comment (deads2k) [reply OWNERS:13]: at your discretion.

**Comment by deads2k** [comment]
gotta start somewhere.

**Pull request merged**